### PR TITLE
Resolve "1408 - upload-terminology command does not work for DSTU3. Fix this!"

### PR DESCRIPTION
**Pull request by dmuylwyk**
Fixed!

### CODE REVIEW

File: hapi-fhir-cli/hapi-fhir-cli-api/src/main/java/ca/uhn/fhir/cli/UploadTerminologyCommand.java [hunk 1]
```diff
@@ -94,12 +93,28 @@ public void run(CommandLine theCommandLine) throws ParseException {
...
 		ourLog.info("Beginning upload - This may take a while...");
-		IBaseParameters response = client
-			.operation()
-			.onType(CodeSystem.class)
-			.named(UPLOAD_EXTERNAL_CODE_SYSTEM)
-			.withParameters(inputParameters)
-			.execute();
+
+		IBaseParameters response;
+		switch (myFhirCtx.getVersion().getVersion()) {
```
Comment (fil512) [inline hapi-fhir-cli/hapi-fhir-cli-api/src/main/java/ca/uhn/fhir/cli/UploadTerminologyCommand.java]: There are two FhirContext variables in this method.  I recommend removing one of them to eliminate confusion.

File: hapi-fhir-cli/hapi-fhir-cli-api/src/main/java/ca/uhn/fhir/cli/UploadTerminologyCommand.java [hunk 2]
```diff
@@ -94,12 +93,28 @@ public void run(CommandLine theCommandLine) throws ParseException {
...
+			case DSTU3:
```
Comment (fil512) [inline hapi-fhir-cli/hapi-fhir-cli-api/src/main/java/ca/uhn/fhir/cli/UploadTerminologyCommand.java]: In place of the switch, what about grabbing the class this way:
`myFhirCtx.getResourceDefinition("CodeSystem").getImplementingClass()`

File: hapi-fhir-cli/hapi-fhir-cli-api/src/main/java/ca/uhn/fhir/cli/UploadTerminologyCommand.java (hunk 1 above)
Comment (dmuylwyk) [reply hapi-fhir-cli/hapi-fhir-cli-api/src/main/java/ca/uhn/fhir/cli/UploadTerminologyCommand.java]: I noticed this, too, but got distracted. Will fix.

File: hapi-fhir-cli/hapi-fhir-cli-api/src/main/java/ca/uhn/fhir/cli/UploadTerminologyCommand.java (hunk 2 above)
Comment (dmuylwyk) [reply hapi-fhir-cli/hapi-fhir-cli-api/src/main/java/ca/uhn/fhir/cli/UploadTerminologyCommand.java]: Ooh, I like this!

File: hapi-fhir-cli/hapi-fhir-cli-api/src/main/java/ca/uhn/fhir/cli/UploadTerminologyCommand.java (hunk 1 above)
Comment (dmuylwyk) [reply hapi-fhir-cli/hapi-fhir-cli-api/src/main/java/ca/uhn/fhir/cli/UploadTerminologyCommand.java]: Fixed.

File: hapi-fhir-cli/hapi-fhir-cli-api/src/main/java/ca/uhn/fhir/cli/UploadTerminologyCommand.java (hunk 2 above)
Comment (dmuylwyk) [reply hapi-fhir-cli/hapi-fhir-cli-api/src/main/java/ca/uhn/fhir/cli/UploadTerminologyCommand.java]: Fixed. Thanks for this, @fil512.

**Pull request merged**